tikv/raftstore/util.go

```diff
@@ -456,3 +468,11 @@ func checkPeerID(req *raft_cmdpb.RaftCmdRequest, peerID uint64) error {
 	}
 	return errors.Errorf("mismatch peer id %d != %d", peer.Id, peerID)
 }
+
+func CloneMsg(origin, cloned proto.Message) error {
```

Comment:
I think cloned should be a pointer type?

Reply:
It's an interface, the interface contains a pointer.

Reply:
got it